Util/AdminObjectAclManipulator.php

```diff
@@ -271,8 +271,21 @@ protected function buildForm(AdminObjectAclData $data, FormBuilderInterface $for
                     'attr'     => $attr,
                 );
             }
-
-            $formBuilder->add($key, new AclMatrixType(), array('permissions' => $permissions, 'acl_value' => $aclValue));
+            //@todo Remove when dropping Symfony <2.8 support
+            $aclMatrixType = new AclMatrixType();
```

Comment:
This is not the good method. Here, you are instantiating a class sometime for nothing.

Please use Symfony `AbstractType` FQDN on `method_exists` instead.